components/intake.py

```diff
@@ -11,6 +11,7 @@ class Intake:
     speed = magicbot.will_reset_to(0.0)
 
     def setup(self) -> None:
+        self.intake_motor.restoreFactoryDefaults()
```

Comment:
This seems unrelated to shooter?

Reply:
yes, the intake would sometimes run backwards and this seemed to fix it, should i make a seperate pull request?

Reply:
A separate PR would be good. It seems like we should just always call `setInverted` on our motor controllers though?